src/test/scala/uk/gov/nationalarchives/pentaho/QueryManagerSpec.scala

```diff
@@ -13,13 +13,13 @@ class QueryManagerSpec extends AnyWordSpec with Matchers with MockitoSugar {
     "return a Success with a count value" in {
       val sparqlQuery =
         "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#> SELECT ?policy ?label WHERE { ?policy rdfs:label ?label . }"
-      val result = QueryManager.executeQuery(sparqlQuery, "example.ttl")
+      val result = QueryManager.executeQuery(sparqlQuery, "src/test/resources", "example", ".ttl")
```

Comment:
Again, I am just wondering about the relative path being encoded here? Would it be possible instead to retrieve the resource via the classpath?

Reply:
Updated.